cmd/dao_tests/account_test.go

```diff
@@ -0,0 +1,169 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"context"
+	"database/sql"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func createAccount() *models.Account {
+	return &models.Account{
+		OrgID:         "2",
+		AccountNumber: sql.NullString{String: "100", Valid: true},
+	}
+}
+
+func createAccountWithNullAccountNumber() *models.Account {
+	return &models.Account{
+		OrgID:         "2",
+		AccountNumber: sql.NullString{},
+	}
+}
+
+func setupAccount(t *testing.T) (dao.AccountDao, context.Context) {
+	setup()
+	ctx := identity.WithTenant(t, context.Background())
+	accDao, err := dao.GetAccountDao(ctx)
+	if err != nil {
+		panic(err)
+	}
+	return accDao, ctx
+}
+
+func teardownAccount(_ *testing.T) {
+	teardown()
+}
+
+func TestCreateAccount(t *testing.T) {
+	accDao, ctx := setupAccount(t)
+	defer teardownAccount(t)
+	acc := createAccount()
+	err := accDao.Create(ctx, acc)
+	if err != nil {
+		t.Errorf("create failed: %v", err)
+		return
+	}
```

Comment:
Just a nitpick: For dao errors, you can simply call panic because it will only happen when imports are incorrect.